test/unit/presenters/plan_presenters_test.rb

```diff
@@ -9,8 +9,8 @@ class PlanPresentersTest < ActiveSupport::TestCase
     assert_same_elements plans.order(created_at: :desc), presenter({ sort: 'created_at', direction: 'desc' }).plans
```

Comment:
This test is wrong. See that plans are ordered by created_at and yet in your implementation you reordered by name. So this test is a false positive.
When sorting matter we should use `assert_equal` instead.